extras/splat/ply-reader.js

```diff
@@ -12,25 +12,32 @@ const dataTypeMap = new Map([
     ['double', Float64Array]
 ]);
 
-class PlyProperty {
-    type;
-
-    name;
-
-    storage;
-
-    byteSize;
-}
-
-class PlyElement {
-    name;
-
-    count;
-
-    properties;
-}
-
-// asynchronously read a ply file data
+/**
+ * @typedef {Int8Array|Uint8Array|Int16Array|Uint16Array|Int32Array|Uint32Array|Float32Array|Float64Array} DataType
+ */
+
+/**
+ * @typedef {object} PlyProperty
```

Comment:
do we even use these types anywhere?

Reply:
yep, `PlyElement` is used once in `ply-reader.js` and three times in `splat-data.js`:

```js
class SplatData {
    /** @type {import('./ply-reader').PlyElement[]} */
    elements;

    /** @type {import('./ply-reader').PlyElement} */
    vertexElement;

    /**
     * @param {import('./ply-reader').PlyElement[]} elements - The elements.
     * @param {boolean} [performZScale] - Whether to perform z scaling.
     */
    constructor(elements, performZScale = true) {
```